<Type Name="EventDeliveryFailedException" FullName="System.Workflow.Activities.EventDeliveryFailedException">
  <Metadata><Meta Name="ms.openlocfilehash" Value="58706e006701f6f1a1bc516ebb4d607e3dfaac3b" /><Meta Name="ms.sourcegitcommit" Value="88014e1c5440e3df4f66ef04393854d15b1fd534" /><Meta Name="ms.translationtype" Value="MT" /><Meta Name="ms.contentlocale" Value="ru-RU" /><Meta Name="ms.lasthandoff" Value="09/05/2019" /><Meta Name="ms.locfileid" Value="70578785" /></Metadata><TypeSignature Language="C#" Value="public sealed class EventDeliveryFailedException : SystemException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable sealed beforefieldinit EventDeliveryFailedException extends System.SystemException" />
  <TypeSignature Language="DocId" Value="T:System.Workflow.Activities.EventDeliveryFailedException" />
  <TypeSignature Language="VB.NET" Value="Public NotInheritable Class EventDeliveryFailedException&#xA;Inherits SystemException" />
  <TypeSignature Language="C++ CLI" Value="public ref class EventDeliveryFailedException sealed : SystemException" />
  <TypeSignature Language="F#" Value="type EventDeliveryFailedException = class&#xA;    inherit SystemException" />
  <AssemblyInfo>
    <AssemblyName>System.Workflow.Activities</AssemblyName>
    <AssemblyVersion>3.0.0.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.SystemException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute FrameworkAlternate="netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
      <AttributeName>System.Obsolete("The System.Workflow.* types are deprecated.  Instead, please use the new types from System.Activities.*")</AttributeName>
    </Attribute>
    <Attribute>
      <AttributeName>System.Serializable</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary><span data-ttu-id="96aef-101">Исключение, которое создается при невозможности доставки возникшего на узле события экземпляру рабочего процесса.</span><span class="sxs-lookup"><span data-stu-id="96aef-101">The exception that is thrown when an event that is raised from the host cannot be delivered to the workflow instance.</span></span> <span data-ttu-id="96aef-102">Как правило, событие возникает из класса <see cref="T:System.Workflow.Activities.ExternalDataExchangeService" /> экземпляра рабочего процесса.</span><span class="sxs-lookup"><span data-stu-id="96aef-102">Typically the event is raised from an <see cref="T:System.Workflow.Activities.ExternalDataExchangeService" /> on a workflow instance.</span></span> <span data-ttu-id="96aef-103">Этот класс не наследуется.</span><span class="sxs-lookup"><span data-stu-id="96aef-103">This class cannot be inherited.</span></span></summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
  
> [!NOTE]
>  [!INCLUDE[DeprecatedContent](~/includes/deprecatedcontent-md.md)]  
  
 <span data-ttu-id="96aef-104">Следующая строка добавляется в журнал событий при возникновении этого исключения: событие "{1}" в интерфейсе типа "{0}" для идентификатора{2}экземпляра "" не может быть доставлено.</span><span class="sxs-lookup"><span data-stu-id="96aef-104">The following string is added to the event log when this exception is thrown: Event "{1}" on interface type "{0}" for instance id "{2}" cannot be delivered.</span></span>  
  
 <span data-ttu-id="96aef-105">При использовании рабочего потока конечного автомата можно получить исключение с сообщением: "Очередь "{0}"' не включена".</span><span class="sxs-lookup"><span data-stu-id="96aef-105">When using a state machine workflow, you might get an exception with the message "Queue '{0}' is not enabled."</span></span> <span data-ttu-id="96aef-106">Это происходит, когда при текущем состоянии конечного автомата не известно, как обработать определенное событие.</span><span class="sxs-lookup"><span data-stu-id="96aef-106">This happens when the current state of the state machine does not know how to handle a specific event.</span></span> <span data-ttu-id="96aef-107">Например, если какое-либо состояние, отличное от текущего, содержит действие <xref:System.Workflow.Activities.EventDrivenActivity>, содержащее действие <xref:System.Workflow.Activities.HandleExternalEventActivity>, которое представляется очередью "{0}".</span><span class="sxs-lookup"><span data-stu-id="96aef-107">For example, when some state other than the current state contains the <xref:System.Workflow.Activities.EventDrivenActivity> that contains the <xref:System.Workflow.Activities.HandleExternalEventActivity> that is represented by the queue '{0}'.</span></span>  
  
 ]]></format>
    </remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Workflow.Activities</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary><span data-ttu-id="96aef-108">Инициализирует новый экземпляр класса <see cref="T:System.Workflow.Activities.EventDeliveryFailedException" />.</span><span class="sxs-lookup"><span data-stu-id="96aef-108">Initializes a new instance of the <see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /> class.</span></span></summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public EventDeliveryFailedException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; EventDeliveryFailedException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Workflow.Activities</AssemblyName>
        <AssemblyVersion>3.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary><span data-ttu-id="96aef-109">Инициализирует новый экземпляр класса <see cref="T:System.Workflow.Activities.EventDeliveryFailedException" />.</span><span class="sxs-lookup"><span data-stu-id="96aef-109">Initializes a new instance of the <see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /> class.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public EventDeliveryFailedException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; EventDeliveryFailedException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new System.Workflow.Activities.EventDeliveryFailedException : string -&gt; System.Workflow.Activities.EventDeliveryFailedException" Usage="new System.Workflow.Activities.EventDeliveryFailedException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Workflow.Activities</AssemblyName>
        <AssemblyVersion>3.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="96aef-110">Сообщение об ошибке с объяснением причины исключения.</span><span class="sxs-lookup"><span data-stu-id="96aef-110">An error message that explains the reason for the exception.</span></span></param>
        <summary><span data-ttu-id="96aef-111">Инициализирует новый экземпляр класса <see cref="T:System.Workflow.Activities.EventDeliveryFailedException" />, используя сообщение об ошибке, в котором объясняется причина исключения.</span><span class="sxs-lookup"><span data-stu-id="96aef-111">Initializes a new instance of the <see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /> class by using the error message that explains the reason for the exception.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public EventDeliveryFailedException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; EventDeliveryFailedException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.Workflow.Activities.EventDeliveryFailedException : string * Exception -&gt; System.Workflow.Activities.EventDeliveryFailedException" Usage="new System.Workflow.Activities.EventDeliveryFailedException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Workflow.Activities</AssemblyName>
        <AssemblyVersion>3.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="96aef-112">Сообщение об ошибке с объяснением причины исключения.</span><span class="sxs-lookup"><span data-stu-id="96aef-112">An error message that explains the reason for the exception.</span></span></param>
        <param name="innerException"><span data-ttu-id="96aef-113">Объект <see cref="T:System.Exception" />, вызвавший это исключение.</span><span class="sxs-lookup"><span data-stu-id="96aef-113">The <see cref="T:System.Exception" /> that caused this exception.</span></span></param>
        <summary><span data-ttu-id="96aef-114">Инициализирует новый экземпляр класса <see cref="T:System.Workflow.Activities.EventDeliveryFailedException" />, используя сообщение об ошибке, в котором объясняется причина исключения, и объект <see cref="T:System.Exception" />, вызвавший это исключение.</span><span class="sxs-lookup"><span data-stu-id="96aef-114">Initializes a new instance of the <see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /> class by using the error message that explains the reason for the exception and the <see cref="T:System.Exception" /> that caused this exception.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
  </Members>
</Type>
